fastlane/Fastfile

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+require './script/build.rb'
+require './script/config.rb'
+
+default_platform(:android)
+
+platform :android do
+  before_all do
+    ENV['SLACK_URL'] = Config::SLACK_URL
+    Config.sanity_check
+  end
+
+  desc 'Execute staging deployment'
```

Comment:
and Production deployment? 🤔

Reply:
Supply will be done in a different PR.

Do we need to upload production to Firebase?

Reply:
yes we need it when releasing so the client can try it out too, currently, we are sending APK, which is equivalent but less automation 🤷‍♂️

Reply:
Should it be Production debug?

Reply:
yes for this purpose 🤔 actually we should think of it in a more versatile way I think...because later you can add another lane that uploads the build to Play Store (productionRelease) right?

Reply:
Yes, there will be a lane for production release that goes straight into Play Store with the help of the `supply` plugin. 

I feel that production builds should use the testers features exposed via the Play Store though. Alpha and Beta channel.

We will need an initiatives about the documentation for setting up a project in Play Store too.